backend/pretix/__init__.py

```diff
@@ -383,4 +384,38 @@ def get_user_tickets(conference: Conference, email: str):
         },
     )
 
+    response.raise_for_status()
+
+    return response.json()
+
+
+def get_user_ticket(conference: Conference, email: str, id: str):
+
+    # TODO: filter by orderposition in the PretixAPI
```

Comment:
what is this todo? what is missing?

Reply:
Would be nice to add this thing to the API directly so I don't have to fetch all the tickets to get only one...